core/src/test/scala/unit/kafka/log/LogCleanerManagerTest.scala

```diff
@@ -53,7 +53,7 @@ class LogCleanerManagerTest extends Logging {
   val logConfig: LogConfig = new LogConfig(logProps)
   val time = new MockTime(1400000000000L, 1000L)  // Tue May 13 16:53:20 UTC 2014 for `currentTimeMs`
   val offset = 999
-  val producerStateManagerConfig = new ProducerStateManagerConfig(kafka.server.Defaults.ProducerIdExpirationMs, false)
+  val producerStateManagerConfig = new ProducerStateManagerConfig(org.apache.kafka.server.config.Defaults.PRODUCER_ID_EXPIRATION_MS, false)
```

Comment:
Can we import `org.apache.kafka.server.config.Defaults`?

Reply:
I'll stop pointing it for each file but let's import the class.

Reply:
done